models/inline_strconv_parse_test.go

```diff
@@ -0,0 +1,69 @@
+package models_test
+
+import (
+	"strconv"
+	"testing"
+	"testing/quick"
+
+	"github.com/influxdata/influxdb/models"
+)
+
+func TestParseIntBytesEquivalenceFuzz(t *testing.T) {
+	f := func(b []byte, base int, bitSize int) bool {
```

Comment:
Instead of generating random byte slices, what would you think about doing this:

```go
var buf []byte
f := func(n int64, base int) bool {
    buf = strconv.AppendInt(buf[:0], n, base)
    // fail on errors to either strconv.ParseInt or models.ParseIntBytes
    // ...
}

values := func(v []reflect.Values, r *rand.Rand) {
    v[0] = reflect.ValueOf( int64(r.Uint32())<<32 | int64(r.Uint32()) )
    v[1] = reflect.ValueOf( r.Intn(35)+2 )
}
```

`values` can then be used to generate known-good inputs to `f`, and doing a similar approach for `ParseFloatBytes`